typing/oprint.ml

```diff
@@ -100,8 +100,21 @@ let print_out_value ppf tree =
     | Oval_nativeint i -> fprintf ppf "%nin" i
     | Oval_float f -> pp_print_string ppf (float_repres f)
     | Oval_char c -> fprintf ppf "%C" c
-    | Oval_string s ->
-        begin try fprintf ppf "%S" s with
+    | Oval_string (s, maxlen, kind) ->
+       begin try
+               let len = String.length s in
+                fprintf ppf "%s%S%s"
+                  (match kind with
+                  | Ostr_string -> ""
+                  | Ostr_bytes -> "Bytes.of_string ")
+                  (if len > maxlen then String.sub s 0 maxlen else s)
+                  (if len > maxlen then
+                      Printf.sprintf
+                        "... (* string stripped from %d to %d chars *)"
```

Comment:
I don't think "stripped" is the right verb. @dra27 @mshinwell @lpw25 or any native English speaker, could you advise?